documents/Libraries/sxpbrlib/sxpbrlib_defs.mtlx

```diff
@@ -182,4 +182,48 @@
     <input name="outerConeAngle" type="float" doc="Outer cone angle."/>
   </nodedef>
 
+  <!-- <standardsurface> -->
```

Comment:
As discussed with Arnold team in London, we are making the standardsurface ubershader a member of the SX PBR library, instead of having it as adskSurface in a separate adsk library. In another task I will remove the adskSurface, and I think it's also time to implement the noise nodes in the standard MX library instead of having separate noise nodes in the adsk library. When it's done we can remove the adsk library.

Reply:
Sounds good.